charts/hono/templates/hono-service-device-registry/hono-service-device-registry-secret.yaml

```diff
@@ -63,6 +63,7 @@ stringData:
           insecurePortEnabled: true
           insecurePortBindAddress: "0.0.0.0"
           {{- end }}
+          deviceIdPattern: "^[a-zA-Z0-9-_\\.\\:]++$"
```

Comment:
What's the purpose of the second `+` before the `$`?

Reply:
Good question, none, I guess :). Didn't pay attention here when adopting it from Hono, where it's also superfluous.